src/Microsoft.Diagnostics.Monitoring.WebApi/OptionsDisplayStrings.resx

```diff
@@ -157,6 +157,30 @@
     <value>Buffer size used when copying data from an egress callback returning a stream to the egress callback that is provided a stream to which data is written.</value>
     <comment>The description provided for the CopyBufferSize parameter on all egress provider options.</comment>
   </data>
+  <data name="DisplayAttributeDescription_ConsoleOptions_FormatterName" xml:space="preserve">
+    <value>Name of the log message formatter to use. Defaults to 'Simple'.</value>
+    <comment>The description provided for the FormatterName parameter on ConsoleOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_ConsoleOptions_FormatterOptions" xml:space="preserve">
```

Comment:
All of the ones that are `DisplayAttributeDescription_ConsoleOptions_FormatterOptions`* probably should be `DisplayAttributeDescription_ConsoleFormatterOptions`*

Reply:
Does this also imply that I should rename the FormatterOptions class to be ConsoleFormatterOptions?

Reply:
I think what's going to happen is that FormatterOptions is flattened into three new classes: ConsoleFormatterOptions, JsonFormatterOptions, and SimpleFormatterOptions.